bootstrap/kubeadm/internal/controllers/kubeadmconfig_controller_test.go

```diff
@@ -2193,6 +2194,115 @@ func TestKubeadmConfigReconciler_ResolveUsers(t *testing.T) {
 	}
 }
 
+func TestAddNodeUninitializedTaint(t *testing.T) {
+	dummyTaint := corev1.Taint{
+		Key:    "dummy-taint",
+		Value:  "",
+		Effect: corev1.TaintEffectNoSchedule,
+	}
+
+	tests := []struct {
+		name              string
+		nodeRegistration  *bootstrapv1.NodeRegistrationOptions
+		kubernetesVersion semver.Version
+		isControlPlane    bool
+		wantTaints        []corev1.Taint
+	}{
+		{
+			name: "for control plane with version < v1.24.0, if taints is nil it should add the uninitialized and the master taints",
```

Comment:
```suggestion
			name: "for control plane with version < v1.24.0, if taints is nil it should add the uninitialized and the master taint",
```
nit: please check below for similar cases